### PR TITLE
Global task button component

### LINKED ISSUE #23: Create task button component

### PR DESCRIPTION
**Pull request by thakurtushar02**
Closes #23

### CODE REVIEW

File: src/components/global/Task/Task.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          {taskName}
+        </span>
+        <RightChevron handleOnClick={handleIconClick} isRotated={isExpanded} />
+      </div>
+
+      <div
+        className={"task-details-container" + (isExpanded ? "--expanded" : "")}
+      >
+        <div className="subtask-list-container">
+          {subtasks.map((s) => (
```
Comment (M-Stable) [inline src/components/global/Task/Task.jsx:31]: You should have learnt from your Persuall reading that you should use more meaningful variable names to improve readability.

File: src/stylesheets/components/global/task/task.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  background: transparent;
+  border: 5px solid white;
+  border-radius: 50%;
+
+  box-sizing: border-box;
+  box-shadow: 0px 0px 6px 1px rgba(255, 236, 236, 0.25);
+}
+
+.task-checkbox--checked {
+  background: #c4c4c4;
```
Comment (franc-hoon) [inline src/stylesheets/components/global/task/task.scss]: put into variable?

**Review by franc-hoon — CHANGES_REQUESTED** [review]
Great work! Just some minor changes 👍

File: src/stylesheets/components/global/task/task.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
+.task-container,
+.task-container--expanded {
+  position: relative;
+  width: 100%;
+}
+
+.task-container {
+  height: $task-header-height + 4px;
```
Comment (max-gurr) [inline src/stylesheets/components/global/task/task.scss:8]: does this work without wrapping it in `calc()`?

**Review by max-gurr — APPROVED** [review]
Nice work, looks good to me!

File: src/stylesheets/components/global/task/task.scss (hunk 3 above)
Comment (thakurtushar02) [reply src/stylesheets/components/global/task/task.scss:8]: Yeah, surprisingly

**Review by YungiKim1999 — APPROVED** [review]

**Review by franc-hoon — APPROVED** [review]
LokTM 👍

**Pull request merged**